torch/linalg/__init__.py

```diff
@@ -140,3 +140,43 @@
     >>> LA.norm(m[0, :, :]), LA.norm(m[1, :, :])
     (tensor(3.7417), tensor(11.2250))
 """)
+
+tensorinv = _add_docstr(_linalg.linalg_tensorinv, r"""
+linalg.tensorinv(input, ind=2, *, out=None) -> Tensor
+
+Computes a tensor ``input_inv`` such that ``tensordot(input_inv, input, ind) == I_n``,
+where ``I_n`` is the n-dimensional identity tensor and ``n`` is equal to ``input.ndim``.
+The resulting tensor ``input_inv`` has the shape equal to ``input.shape[ind:] + input.shape[:ind]``.
+
+Supports real and, only on the CPU, complex inputs.
+
+.. note:: If :attr:`input` is not a 'square' tensor, meaning it does not satisfy the requirement
+          ``prod(input.shape[ind:]) == prod(input.shape[:ind])`` or if :attr:`input` is not invertible,
+          then a RuntimeError will be thrown.
+
+.. note:: If :attr:`input` is a 2-dimensional tensor and ``ind=1``, then this function computes matrix inverse.
```

Comment:
"If" -> "When" 

", then this function compute matrix inverse" -> "this function computes the (multiplicative) inverse of :attr:\`input\`, equivalent to calling :func:\`torch.inverse\`."

(That sample func reference may need a formatting fix.)